### ISSUE TITLE
can't perform backups, fc remnants

### ISSUE THREAD

**Issue by tsiflimagas** [machiav3lli/oandbackupx#23]
Hi, the new update(0.9.12) improved the force close issue by much, though there are still some, but broke the backup function. The logs for the fc issue are those (basically two errors xD) https://del.dog/crilegutul
The error for the backup issue is this
2020/05/14 - 04:48:25: /system/bin/sh: <stdin>[1]: rsync: inaccessible or not found.
Also, I may refer to some ui stuff, the splash screen doesn't quite show up, it flashes quickly and then fades out. Pull down to refresh also doesn't work very good, icon gets dragged back immediately after pulled down in a choppy motion and I'm not sure if refreshing is actually done. Other than these great work, keep it up😃

**Comment by talk2g** [comment]
i too m having the same issue, albeit on the device that worked fine in the last update (This device didnt have the fc issue earlier)
However on the device that was facing fc issues, the number of fc have been far less than before and it can perform backups perfectly even in the new update.
TLDR- cant backup apps on the atest version in the device that worked fine until now.
 can backup apps on latest version in the device that faced frequent fc and the number of fc have been reduced too.
EDIT- clean installed and deleted backup directly before testing on both devices.

**Comment by talk2g** [comment]
![Screenshot_20200514-110350393](https://user-images.githubusercontent.com/23628191/81898081-95adf600-95d5-11ea-9841-a04b0c65ba2f.jpg)
![Uploading Screenshot_20200514-110303292.jpg…]()

**Comment by talk2g** [comment]
![Screenshot_20200514-110303292](https://user-images.githubusercontent.com/23628191/81898144-baa26900-95d5-11ea-98a7-a6895c40d84c.jpg)

**Comment by machiav3lli** [comment]
Ah ok, so rsync isn't bundled in your ROMs, I'll revert back to cp and post a hotfix release. Thanks
